src/components/MapPopover.vue

```diff
@@ -62,6 +62,10 @@ export default {
       }
     return false;
     },
+    setTooltipCoords: function(x, y){
```

Comment:
You should not change values of props in a component, either change it on the parent component or change the data "tStyle". Take at look at the watch routine.

Reply:
Yep algood I will switch this to FloatingDialog now.